b_lambda_layer_common/source/python/b_lambda_layer_common/cache/data_cache.py

```diff
@@ -0,0 +1,80 @@
+import logging
+from typing import Any, Callable, Set, Union
+
+logger = logging.getLogger(__name__)
+
+try:
+    from b_lambda_layer_common.exceptions.container.internal_error import InternalError
+    from b_lambda_layer_common.ssm.refreshable import Refreshable
```

Comment:
Importing from ssm directory does not make any sense. Write a TODO comment that "Refreshable" class should be moved somewhere else.